Базовая стилизация всех страниц

VladissGernet opened the pull request:
---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/34/user/1981559/tasks/6)

:boom: https://htmlacademy-htmlcss.github.io/1981559-sedona-34/4/

VladissGernet committed:
Исправлен недочет в фильтрации.

VladissGernet committed:
Добавил файл стилей и нормализатор

VladissGernet committed:
добавляю шрифты

VladissGernet committed:
изменение ссылок

VladissGernet committed:
добавил css текста в хедер, подправил недочеты

VladissGernet committed:
добавил пропущенные классы

VladissGernet committed:
css text для index.html готов

VladissGernet committed:
Шрифты в каталоге

VladissGernet committed:
добавил font-weight

VladissGernet committed:
Начал редактировать фильтр

VladissGernet committed:
Исправил шрифт во всех кнопках

Всего-лишь одная строка кода.

VladissGernet committed:
Добавил стили для всего текста

VladissGernet committed:
Добавил настройки цвета шрифта для контраста

VladissGernet committed:
Update styles.css

Dima-Dim commented on styles/styles.css:
```diff
@@ -0,0 +1,316 @@
+  text-transform: uppercase;
+}
+
+newsletter-words {
```
`.` пропущена.

Dima-Dim commented on index.html:
```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Седона</title>
+    <link rel="stylesheet" href="normalize.css">
```
Не верно указан путь до стилей `normalize.css`.
Проблему можно видеть в девтулзах опубликованной версии https://htmlacademy-htmlcss.github.io/1981559-sedona-34/4/

VladissGernet replied:
Удалил его вообще, так как в 5 разделе говорят, что не стоит его использовать в проекте.

VladissGernet committed:
Добавлены страхуещие верстку строки

VladissGernet committed:
Пропущенную точку исправил

VladissGernet committed:
Нормализатор подключен

VladissGernet committed:
Удалил нормализатор

В 5 разделе говорится, что его не следует использовать в заданиях.

VladissGernet committed:
Убрал лишние 400, подправил селекторы

Исправлен шрифт в селекторах выбора сортировки по ценнику и количеству гостиниц на странице
